### PR TITLE
Resolve issues raised on beta-2

### LINKED ISSUE #5234: Issues observed on ubuntu beta.2
#### Account Management
- On registering multi-signature account, I get a NaN when I try to change the required signatures even though I have on member as mandatory and the other as optional (it’s hard to reproduce)
image

- I am able to continue to the summary page on register multi-signature if I make the required signatures field empty and I make all members optional.
image

- I get a weird error when I don’t have token and I want to register a multi-signature account
image

#### Validator
- I am unable to see the transaction params on the summary page of a sign change of commission transaction
image

#### Wallet Page

- On request tokens, address gets clamped. I was expecting to see a tooltip that shows me the full address.
image

- Event table does not get updated after a transaction is done.

### Which version(s) does this affect? (Environment, OS, etc...)
- Ubuntu

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #5234

### How was it solved?

- [x] Fix issues raised on #5234
- [x] Added a no token modal notification on transaction triggers when an account has no token balance
- [x] Resolve unit tests

### How was it tested?
- Test through the issues raised on #5234.

### CODE REVIEW

File: src/modules/wallet/components/overview/overview.js [hunk 1]
```diff
@@ -125,8 +125,11 @@ const Overview = ({ isWalletRoute, history }) => {
             )}
           </div>
           <div className={`${grid['col-xs-3']} ${grid['col-md-3']} ${grid['col-lg-3']}`}>
-            <DialogLink component="send">
-              <PrimaryButton disabled={!hasTokenWithBalance}>{t('Send')}</PrimaryButton>
+            <DialogLink
+              data={{ message: 'There are no tokens to send at this time.' }}
```
Comment (ManuGowda) [inline src/modules/wallet/components/overview/overview.js]:
```suggestion
              data={{ message: 'There are no tokens to send at this moment.' }}
```

Comment (ManuGowda) [reply src/modules/wallet/components/overview/overview.js]: Also this is missing translation?

File: src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+function NoTokenBalance({ onRequestToken }) {
+  const { t } = useTranslation();
+  const location = useLocation();
+  const queryParams = new URLSearchParams(location.search);
+  const message = queryParams.get('message');
+
+  return (
+    <div className={styles.errorWrapper}>
+      <Illustration className={styles.emptyTokenIllustration} name="emptyTokensIllustration" />
+      <p>{message || t('There are no tokens to display for this account at this time.')}</p>
```
Comment (ManuGowda) [inline src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.js]:
```suggestion
      <p>{message || t('There are no tokens to display for this account at this moment.')}</p>
```

File: src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js [hunk 3]
```diff
@@ -38,8 +40,15 @@ const ValidatorActionButton = ({ address, isValidator }) => {
...
   }
 
+  const hasTokenBalances = tokenBalances.data?.data?.some(
+    ({ availableBalance }) => !!BigInt(availableBalance)
+  );
+
   return (
-    <DialogLink component="registerValidator">
+    <DialogLink
+      data={{ message: 'There are no tokens to register a validator at this time.' }}
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js]:
```suggestion
      data={{ message: 'Token balance is not enough to register a validator.' }}
```

File: src/modules/pos/validator/components/ValidatorProfile/ValidatorStakeButton.js [hunk 4]
```diff
@@ -24,7 +24,10 @@ function ValidatorStakeButton({ address, isBanned, currentAddress, isDisabled })
   const isEdit = validatorStake || staking[address];
 
   return (
-    <DialogLink component="editStake">
+    <DialogLink
+      data={{ message: 'There are no tokens available to stake a validator at this time.' }}
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorProfile/ValidatorStakeButton.js]:
```suggestion
      data={{ message: 'Token balance is not enough to stake a validator.' }}
```

Comment (ManuGowda) [reply src/modules/pos/validator/components/ValidatorProfile/ValidatorStakeButton.js]: missing translation

File: src/modules/account/const/accountMenu.js [hunk 5]
```diff
@@ -23,7 +29,8 @@ export const accountMenu = (authData, isHW, hasNetworkError, isLoadingNetwork) =
     label: 'Add new account',
   },
   {
-    component: 'multiSignature',
+    component: hasAvailableTokenBalance ? 'multiSignature' : 'noTokenBalance',
+    data: { message: 'There are no tokens to register a multisignature account at this time.' },
```
Comment (ManuGowda) [inline src/modules/account/const/accountMenu.js]:
```suggestion
    data: { message: 'Token balance is not enough to register a multisignature account.' },
```

File: src/modules/account/components/AccountMenuListing/AccountMenuListing.js [hunk 6]
```diff
@@ -8,35 +8,45 @@ import { Link } from 'react-router-dom';
...
   return (
     <ul className={className}>
-      {accountMenu(authData, isHW, hasNetworkError, isLoadingNetwork).map(
-        ({ path, icon, label, component, isHidden }) =>
+      {accountMenu({
+        authData,
+        isHW,
+        hasNetworkError,
+        isLoadingNetwork,
+        hasAvailableTokenBalance: tokenBalances.data?.data?.length > 0,
```
Comment (ManuGowda) [inline src/modules/account/components/AccountMenuListing/AccountMenuListing.js]:
```suggestion
        hasAvailableTokenBalance: tokenBalances.data?.data?.length > 0,
```
This should be a check against available balance > 0? similar to other place where we have `.some( balance greater than 0`?

Comment (ManuGowda) [reply src/modules/account/components/AccountMenuListing/AccountMenuListing.js]: e.g tokenBalances.data?.data?.some(
    ({ availableBalance }) => BigInt(availableBalance) > 0
  );

File: src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.test.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  });
+
+  it('should render custom message', async () => {
+    useLocation.mockReturnValue({ search: '?message=test-message' });
+
+    const wrapper = renderWithRouter(NoTokenBalance, { onRequestToken: mockOnRequestToken });
+
+    expect(
+      wrapper.queryByText('There are no tokens to display for this account at this monent.')
+    ).toBeFalsy();
```
Comment (ManuGowda) [inline src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.test.js:34]: maybe assert `test-message` as well?

File: src/modules/wallet/components/RegisterMultisigForm/index.js [hunk 8]
```diff
@@ -119,6 +119,8 @@ const Form = ({ nextStep, prevState = {}, onNext }) => {
 
   const changeNumberOfSignatures = (e) => {
     const value = e.target.value ? Number(e.target.value) : undefined;
+    // if (value > MAX_MULTI_SIG_MEMBERS) return;
+
```
Comment (ManuGowda) [inline src/modules/wallet/components/RegisterMultisigForm/index.js]: remove?

**Review by ManuGowda — COMMENTED** [review]
LG, few minor comments

File: src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.test.js (hunk 7 above)
Comment (eniolam1000752) [reply src/modules/token/fungible/components/NoTokenBalanceDialog/NoTokenBalance.test.js:34]: it was done below

File: src/modules/account/components/AccountMenuListing/AccountMenuListing.js [hunk 9]
```diff
@@ -8,35 +8,47 @@ import { Link } from 'react-router-dom';
...
     <ul className={className}>
-      {accountMenu(authData, isHW, hasNetworkError, isLoadingNetwork).map(
-        ({ path, icon, label, component, isHidden }) =>
+      {accountMenu({
+        authData,
+        isHW,
+        hasNetworkError,
+        isLoadingNetwork,
+        hasAvailableTokenBalance: tokenBalances.data?.data?.some(
+          ({ availableBalance }) => !!BigInt(availableBalance)
```
Comment (ManuGowda) [inline src/modules/account/components/AccountMenuListing/AccountMenuListing.js]: shouldn't this be > 0 check?

**Review by ManuGowda — DISMISSED** [review]

**Review by ikem-legend — DISMISSED** [review]

**Review by ManuGowda — APPROVED** [review]

**Review by ikem-legend — APPROVED** [review]

**Pull request merged**